### ISSUE TITLE
Source archives on files.minecraftforge.net are (almost) empty

### ISSUE THREAD

**Issue by bothie** [MinecraftForge/MinecraftForge#3236]
Maybe here not best way to report this, but didn't find a better place to report, so, doing it here:

On http://files.minecraftforge.net, at least for versions 1.7.10-10.13.4.1614-1.7.10 and 1.7.10-10.13.4.1564-1.7.10, the source archives contain only the Tutorial Mod, but not the actual MinecraftForge sources. Probably, this issue affects much more version archives, but I didn't check on that.

**Comment by LexManos** [comment]
This is as intended. Setting up the MDK downloads and sets up all necessary libraries including Minecraft and Forge.
The 'src' distributions have not contained the 'source' for quite some time which is why they were renamed to MDK, Mod Development Kit.

**Comment by bothie** [comment]
I still don't know a better place to have this conversation, so I'll ask follow up questions here for now.

I didn't exactly tell so in the first post, because I actually intended for a bug report and not for some kind of support. After the "works as intended" answer, now I actually need support:

Current status: I have a working development environment with the decompiled code of 1.7.10-10.13.4.1614-1.7.10 and everything works as intended.
Goal: I want to keep my development environment but replace that decompiled code by the actual source code.

What I need is a way to obtain the actual source code that was used to create the 1614 jar file. Nothing more, nothing less. How do I obtain that without the need to run gradle and letting it download tons of stuff and do lots of work that I will rm -fr a few minutes later anyways? The last two hours, I tried to understand how the MDK stuff downloads the source, but I didn't get it yet. So, the situation now is: I don't have a problem with working on the decompiled code. But you will have problems with me working on that, because any bugfix I will send you will be based on the decompiled code and not on the original source code.

**Comment by LexManos** [comment]
1) Development for 1.7.10 has stopped, you should stop as well. No bugfixes will be accepted unless its 'This bug happens to 90% of users and ti formats their harddrive and kills their dog' type of bugs.
2) If you ARE working ON Forge then use the git repo not the MDK
3) EITHER way, there is ONLY Decompiled code because Minecraft is a closed source project there is no 'source' to download. We have to decompile it.

TLDR: Stop, read our documentation, and update to 1.10.2.

**Comment by bothie** [comment]
I'm not going to update FTB Infinity to 1.10.2. I'll continue to work with decompiled forge.
